lang/nl.json

```diff
@@ -14,13 +14,24 @@
   "Unmute": "Geluid aan",
   "Playback Rate": "Weergavesnelheid",
   "Subtitles": "Ondertiteling",
-  "subtitles off": "Ondertiteling uit",
-  "Captions": "Ondertiteling",
-  "captions off": "Ondertiteling uit",
+  "subtitles off": "ondertiteling uit",
```

Comment:
And even if it is intentional, I think that in that particular spot, it would be a stylistic choice, for which there is the text-transform css property.